docs/varnish/vcl/parameters.vcl

```diff
@@ -5,7 +5,7 @@
 // https://github.com/ezsystems/ezplatform/blob/master/doc/docker/entrypoint/varnish/parameters.vcl
 
 backend ezplatform {
-    .host = "127.0.0.1";
+    .host = "web"; // Replace with hostname/ip of the application server
```

Comment:
I don't think  these changes are relevant. They look docker specific. So I would recommend leaving them as-is

The paramters.yml file used by docker containers are anyway this one  https://github.com/ezsystems/ezplatform/blob/master/doc/docker/entrypoint/varnish/parameters.vcl

Reply:
Hi Vidar,

As for those changes, they are required to make tests pass properly.
Specifically in https://github.com/ibexa/docker/pull/7/files

However, we should simply refresh that PR to use the file mentioned above, and we should be good to go.